docker-compose.yml

```diff
@@ -2,6 +2,16 @@ version: '3'
 
 services:
 
+  elasticsearch:
+    container_name: edx.devstack.blockstore-elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.4.3
+    ports:
+       - "9200:9200"
+       - "9300:9300"
```

Comment:
I'm not sure that sharing these on the host as 9200/9300 is a good idea - seems like they're likely to conflict? Perhaps use ports like 8255 and 8256 which are close in number to blockstore (8250) ?

Though I guess it's fine to leave this and only change it if there's a problem.

edX doesn't expose elasticsearch to the host at all btw: https://github.com/edx/devstack/blob/db52ce74695bc3d48ed63f1f8e02d071b22d085b/docker-compose.yml#L25-L34

Reply:
Hmm. Having them exposed is useful for interacting with the ES REST API. I'll change them to 8255, 8256.